Dockerfile

```diff
@@ -46,36 +41,17 @@ RUN apk add --update --no-cache git g++ linux-headers
 
 ARG USER
 ARG config_file=environments/docker.toml
-ARG vdb_connect
-
-# setup environment
-ENV GOPATH $HOME/go
-ENV GO111MODULE on
-ENV VDB_PG_CONNECT="$vdb_connect"
 
 # Direct logs to stdout for docker log driver
 RUN ln -sf /dev/stdout /go/src/github.com/makerdao/vulcanizedb/vulcanizedb.log
 
-RUN adduser -Su 5000 $USER
-# container needs to be writable for plugin execution
-RUN chown 5000:5000 /go/src/github.com/makerdao/vulcanizedb
-
-USER $USER
-
-# chown first so dir is writable
-# note: using $USER is merged, but not in the stable release yet
-COPY --chown=5000:5000 --from=builder /go/src/github.com/makerdao/vulcanizedb .
-COPY --chown=5000:5000 --from=builder /go/src/github.com/makerdao/vdb-mcd-transformers /go/src/github.com/makerdao/vdb-mcd-transformers
```

Comment:
also wondering how we're able to remove these lines, specifically why we're able to remove the `--chown`

Reply:
With the plugin already built, we don't need write access to create an .so file or assemble migrations